inspec.yml

```diff
@@ -1,19 +1,18 @@
 name: cis-aws-foundations-baseline
 title: cis-aws-foundations-baseline
 maintainer: MITRE InSpec Team
-copyright: MITRE, 2018
+copyright: MITRE, 2020
 copyright_email: inspec@mitre.org
 license: Apache-2.0
 summary: 'InSpec Profile to validate the CIS AWS Foundations Benchmark v1.1.0'
-version: 1.0.5
+version: 1.2.0
 supports:
   - platform: aws
-
 depends:
   - name: inspec-aws
     git: https://github.com/inspec/inspec-aws.git
```

Comment:
Let's change this to use the url path of archieve master

Reply:
Done